<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QMainWindow" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/New/1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>导入元件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>导出元件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>网络连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>语音输入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>存取速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>屏幕亮度</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>面板导入</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>usb导入</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>面板导入</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>usb导入</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>开关</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>开关</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>快</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>中</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>慢</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>高</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>490</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>中</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_12">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>低</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
